frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -210,6 +211,28 @@
             <template #title><div></div></template>
             <template #runs><div></div></template>
           </omegaup-arena-runs>
+          <omegaup-arena-runs-for-courses
+            v-if="useNewVerdictTable"
```

Comment:
Acá se podrá usar un `v-else`
```suggestion
            v-else
```
?